connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/controller/InboundConnectorRestController.java

```diff
@@ -83,7 +83,8 @@ private List<ActiveInboundConnectorResponse> getActiveInboundConnectors(
 
   private Map<String, Object> getData(ActiveExecutableResponse connector) {
     Map<String, Object> data = Map.of();
-    if (WebhookConnectorExecutable.class.isAssignableFrom(connector.executableClass())) {
+    if (connector.executableClass() != null
```

Comment:
nitpick: for readability, consider assigning `connector.executableClass()` as a variable above line 86, then use that variable in lines 86 and 87.

If you disagree, feel free to ignore 😄

Reply:
Will do